buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -283,9 +300,11 @@ public void freeThreadLocalCache() {
 
     final class PoolThreadLocalCache extends FastThreadLocal<PoolThreadCache> {
         private final AtomicInteger index = new AtomicInteger();
+        final AtomicInteger caches = new AtomicInteger();
```

Comment:
why not :) That said I don't think there will be a difference as we not expect to have many PooledByteBufAllocator instances and so the memory is not an issue.